### PR TITLE
[oneapi] Test: Run only once

### PR DESCRIPTION
**Pull request by wateret**
All the unitests were based on Env(EXECUTOR and BACKENDS). This commit
makes it to run only once.

- Change installation dir `unittest` to `unittest_standalone`
- Fix unittest TestInputReshapingAddModelLoaded
- Running oneapi test is called from `test_ubuntu_runtime_mixed.sh`
  temporarilly

ONE-DCO-1.0-Signed-off-by: Hanjoung Lee <hanjoung.lee@samsung.com>

### CODE REVIEW

**Comment by ragmani** [comment]
Please rebase after merging #1408

**Comment by wateret** [comment]
@hyunsik-yoon @hseok-oh PTAL

File: infra/scripts/test_ubuntu_runtime_mixed.sh [hunk 1]
```diff
@@ -9,6 +9,24 @@ CheckTestPrepared
...
+
+# This test requires test model installation
+pushd ${ROOT_PATH} > /dev/null
+echo
+echo "==== Run oneapi_test begin ===="
+echo
+ONEAPI_TEST_MODEL_INSTALLER=tests/scripts/oneapi_test/install_oneapi_test_nnpackages.sh
+TEST_BIN=Product/out/unittest_standalone/nnfw_api_gtest
+$ONEAPI_TEST_MODEL_INSTALLER --install-dir ${TEST_BIN}_models
+ONERT_LOG_ENABLE=1 ${TEST_BIN}
```
Comment (hyunsik-yoon) [inline infra/scripts/test_ubuntu_runtime_mixed.sh:23]: Is `ONERT_LOG_ENABLE=1` required?

File: tests/nnfw_api/src/ModelTestInputReshaping.cc [hunk 2]
```diff
@@ -35,14 +35,8 @@ TEST_F(TestInputReshapingAddModelLoaded, reshaping_2x2_to_4x2)
...
 
-  // model is already loaded by fixture
-
-  if (!(onlyForCpuBackend(_session) && onlyForLinearExecutor(_session)))
-  {
-    // let's skip this test
-    SUCCEED();
-    return;
-  }
+  ASSERT_EQ(nnfw_set_available_backends(_session, "cpu"), NNFW_STATUS_NO_ERROR);
```
Comment (hyunsik-yoon) [inline tests/nnfw_api/src/ModelTestInputReshaping.cc:38]: :+1:

File: infra/scripts/test_ubuntu_runtime_mixed.sh (hunk 1 above)
Comment (wateret) [reply infra/scripts/test_ubuntu_runtime_mixed.sh:23]: Oops. That should be removed. Thanks!

File: runtime/onert/core/src/compiler/ManualScheduler.cc [hunk 3]
```diff
@@ -29,22 +29,32 @@ namespace onert
...
+    for (auto backend_id : _options.backend_list)
+    {
+      auto backend = resolveBackend(backend_id);
+      if (backend)
+        return backend;
+    }
+    return nullptr;
+  }();
+  assert(!fallback); // There must be at least one fallback
+
```
Comment (hseok-oh) [inline runtime/onert/core/src/compiler/ManualScheduler.cc:54]: @wateret Is it assume that all available backends are loaded before this method is called?

Comment (wateret) [reply runtime/onert/core/src/compiler/ManualScheduler.cc:54]: @hseok-oh Yes. We have `BackendContexts`, that means we have all avaliable backends loaded.

FYI,

`BackendManager` manages dynamic libraries, this is a singleton and may contain backends that are not used for some sessions. It contains the whole set of backends that are used at least once in all the sessions of the current process.

`BackendContexts` is a set of backend contexts that are used for this session(compiler/execution). So `BackendContexts` is a subset(with context) of BackendManager.

**Comment by wateret** [comment]
This will fix #1535

**Pull request merged**